.gitignore

```diff
@@ -0,0 +1,7 @@
+src/org/zaproxy/zap/extension/bruteforce/resources/help_*
+build/build/
+build/log
+build/log1
```

Comment:
This is created every time I run the ant command to deploy the plugin. I put it .gitignore so we don't push build folders to the repo.